smithy-aws-protocol-tests/model/ec2Query/input-lists.smithy

```diff
@@ -85,6 +85,23 @@ apply QueryLists @httpRequestTests([
             ListArgWithXmlName: ["A", "B"]
         }
     },
+    {
+        id: "Ec2ListNestedStructWithList",
+        documentation: "Nested structure with a list member",
+        protocol: ec2Query,
+        method: "POST",
+        uri: "/",
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded"
+        },
+        body: "Action=QueryLists&Version=2020-01-08&NestedWithList.ListArg.member.1=A&NestedWithList.ListArg.member.2=B",
```

Comment:
ec2 doesn't have the `member` segment - it looks like our docs are actually wrong on this

Reply:
Ahh good catch, haven't implemented EC2 protocol yet and missed that difference.